specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,115 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, IPv6, netmask as a part of node
+hardware inventory.
+
+== Use Cases
+
+Tendrl should show the current status of active network interface.
+
+== Proposed change
+* Netifaces package is used to find active network interface in particular
+  node.
+* Add netifaces in requirements.txt and tendrl-node-agent.spec as a dependency.
+* Modify pull_hardware_inventory.py to find active network interfaces details
+  except lo, virbr0, virbr0-nic and update those details into node_inventory
+  list.
+* Create a new class called Network in persistence with respective attribute.
+* Modify manager.py to populate network details from node_inventory list to
+  Network object.
+* Modify persister.py to persist the Network object in etcd.
```

Comment:
Its a bit implementation specific, can you have more like an overview of what you are going to change ?